app/views/tags/index.html.erb

```diff
@@ -0,0 +1,12 @@
+<form action="/tags" method="get">
```

Comment:
The index page should list all of the tags as links to their show page. Please put them in divs for styling purposes in addition to the search function. @drucee @Nova840